text/0517-io-os-reform.md

```diff
@@ -1155,7 +1155,49 @@ RFC recommends they remain unstable.
 #### `stdin`, `stdout`, `stderr`
 [stdin, stdout, stderr]: #stdin-stdout-stderr
 
-> To be added in a follow-up PR.
+The current `stdio` module will be removed in favor of three constructors:
+
+* `stdin` - returns a handle to a **globally shared** to the standard input of
+  the process which is buffered as well. All operations on this handle will
```

Comment:
"buffered as well" here seems to contradict below where it says it won't have BufRead implemented.